pkg/model/constants.go

```diff
@@ -18,4 +18,8 @@ const (
 	PostgresqlPersistentVolumeCapacity = "1Gi"
 	DatabaseSecretUsernameProperty     = "user"
 	DatabaseSecretPasswordProperty     = "password"
+	KeycloakServicePort                = 8443
+	AdminUsernameProperty              = "SSO_ADMIN_USERNAME"    // nolint
```

Comment:
I commented the same snippet on David's PR - how about changing this to `ADMIN_USERNAME` and `ADMIN_PASSWORD`? SSO would suggest that we'll be using the product, whereas KEYCLOAK that we'll be using upstream. I think it would be best to leave it as generic as possible.